docs/src/faq.md

```diff
@@ -28,7 +44,7 @@ Here, `../data/cubicparams.csv` is a parameter definition file that looks someth
 ```
 # Example estimated polynomial parameter
 # First line: linear, quadratic, cubic
-# Lines 2-4: covariance matrix
+# Lines 2-5: covariance matrix
```

Comment:
I think 2-4 is correct?

Reply:
But also, we don't support this do we? passing a distribution to `set_param!`? Is this a very old example?

Reply:
Yes, this is the example that I was going to show @davidanthoff to see if I should take it out completely.  It isn't allowed right now, and I'll be slowly going through all docs to add tests and that'll help us make sure everything is up to date.